### ISSUE TITLE
Make argument building more readable

### ISSUE THREAD

**Issue by raffomania** [svenstaro/wiresmith#28]
The command building in the test utils is a little hard to parse due to the many .arg lines. Consider using [xshell](https://docs.rs/xshell/latest/xshell/index.html) or similar for a more readable command builder.

**Comment by svenstaro** [comment]
I think a more lightweight solution would be to use `.args(&[])`.

**Comment by svenstaro** [comment]
I think the change isn't worth the effort. I did a small experiment it would look like this.

Before:
```rust
    Command::new("podman")
        .arg("run")
        .args(["--name", &format!("consul-{http_port_dc2}")])
        .arg("--replace")
        .arg("--rm")
        .args(["--label", "testcontainer"])
        .args(["--label", &format!("testport={http_port_dc2}")])
        .args(["--network", &format!("wiresmith-{http_port_dc1}")])
        .args(["-p", &format!("{http_port_dc2}:{http_port_dc2}")])
        .arg("docker.io/hashicorp/consul")
        .arg("agent")
        .arg("-dev")
        .args(["--datacenter", "dc2"])
        // This is the part that makes this a federated cluster.
        .args(["-retry-join-wan", &format!("consul-{http_port_dc1}")])
        .args(["-bind", "{{ GetInterfaceIP \"eth0\" }}"])
        .args(["-client", "0.0.0.0"])
        .args(["-http-port", &http_port_dc2.to_string()])
        .args(["-grpc-port", "0"])
        .args(["-grpc-tls-port", "0"])
        .args(["-dns-port", "0"])
        .args(["-serf-lan-port", &port().to_string()])
        .args(["-server-port", &port().to_string()])
        .args(args.clone())
        .stdout(Stdio::null())
        .spawn()
        .expect("Couldn't run Consul binary");
```
```rust
    cmd!(
        &sh,
        "podman run \
         --name consul-{http_port_dc2} \
         --replace \
         --rm \
         --label testcontainer \
         --label testport={http_port_dc2} \
         --network wiresmith-{http_port_dc1} \
         -p {http_port_dc2}:{http_port_dc2} \
         docker.io/hashicorp/consul \
         agent \
         -dev \
         --datacenter dc2 \
         -retry-join-wan consul-{http_port_dc1} \
         -bind {{ GetInterfaceIP \"eth0\" }} \
         -client 0.0.0.0 \
         -http-port {http_port_dc2} \
         -grpc-port 0 \
         -grpc-tls-port 0 \
         -dns-port 0 \
         -serf-lan-port {serf_lan_port_dc2} \
         -server-port {server_port_dc2} \
         {args_str...}"
    )
    .run()
    .expect("Couldn't run Consul binary");
```
and I think that's worse. So let's stick with the current approach.
